src/core/server/models/user/user.ts

```diff
@@ -471,6 +471,17 @@ export interface User extends TenantResource {
    */
   deletedAt?: Date;
 
+  /**
+   * isNew is whether the user is a new user with less approved
+   * comments than the new commenter moderation config allows to
+   * be considered no longer a new user.
+   */
+  isNew: boolean;
```

Comment:
This cannot be stored on the user model document as it is dynamically determined by inspecting the counts on a user.